### PR TITLE
changed package.json & Added randomQuoteFetcher.js

### PR DESCRIPTION
**Pull request by pranjalbhatt1228**
Basic changes in `package.json` and added `randomQuoteFetcher.js`. Added image of anime to quote object.

### CODE REVIEW

**Comment by pranjalbhatt1228** [comment]
Added random anime fetcher also.

File: services/fetcher/randomAnimeFetcher.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        if (res.rank > rankLimit) continue; // rank limit check
+        if (res.title) anime.title = res.title;
+        if (res.url) anime.url = res.url;
+        if (res.synopsis) anime.synopsis = res.synopsis;
+        if (res.genres) {
+          const genre = res.genres.map((g) => g.name);
+          anime.genre = genre;
+        }
+        if (res.rank) anime.rank = res.rank;
+        if (typeof res.airing !== "undefined") anime.airing = res.airing;
```
Comment (AaronStone24) [inline services/fetcher/randomAnimeFetcher.js:27]: why didn't you check this one like others as well?

File: services/fetcher/randomAnimeFetcher.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+const fetchRandomAnimes = async (num) => {
+  let animes = [];
+  while (num--) {
+    const anime = await fetchRandomAnime();
+    if (
+      animes.find((a) => {
+        a === anime;
+      })
```
Comment (AaronStone24) [inline services/fetcher/randomAnimeFetcher.js:48]: `if(find((a) => a===anime))`

File: services/fetcher/randomQuoteFetcher.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    // fetching anime image
+    let animeImageURI = await fetchAnimeImage(response.data.anime || "");
+
+    // fetching character image
+    let characterImageURI = await fetchCharacterImage(
+      response.data.character || ""
+    );
+
+    const quote = { ...response.data, animeImageURI, characterImageURI };
```
Comment (AaronStone24) [inline services/fetcher/randomQuoteFetcher.js:22]: `const quote = { ...response.data, animeImageURI: animeImageURI, characterImageURI: characterImageURI };`

won't it be like this?

File: services/fetcher/randomAnimeFetcher.js (hunk 1 above)
Comment (pranjalbhatt1228) [reply services/fetcher/randomAnimeFetcher.js:27]: because it is a boolean value. checking this like others will affect the working of the variable.

**Review by AaronStone24 — APPROVED** [review]

File: services/fetcher/randomAnimeFetcher.js (hunk 2 above)
Comment (pranjalbhatt1228) [reply services/fetcher/randomAnimeFetcher.js:48]: auto-formatting using prettier ext

File: services/fetcher/randomQuoteFetcher.js (hunk 3 above)
Comment (pranjalbhatt1228) [reply services/fetcher/randomQuoteFetcher.js:22]: if we don't pass key-value pair then the name of the variable automatically becomes the key,

**Pull request merged**